dluhc-component-library/src/components/timetableForm/components/LandingPage.tsx

```diff
@@ -0,0 +1,35 @@
+interface LandingPageProps {
+  continueClick: () => void;
```

Comment:
Sorry also should have spotted this yesterday. I think this should be `onContinueClick` similar to how buttons have `onClick` and inputs have `onChange`